public/locales/en/translation.json

```
@@ -689,6 +690,7 @@
     "fundUpdate": "Update Fund",
     "fundDelete": "Delete Fund",
     "noFundsFound": "No Funds Found",
+    "editFund": "Edit Fund",
```

Biome (2.1.2)

[error] The key editFund was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)